spec/runners/unified/event_subscriber.rb

```diff
@@ -30,8 +30,13 @@ def wanted_events(observe_sensitive = false)
         events
       else
         events.reject do |event|
-          event.respond_to?(:command_name) &&
-            %w(authenticate getnonce saslStart saslContinue).include?(event.command_name)
+          if event.respond_to?(:command_name)
+            command = event.respond_to?(:command) ? event.command : event.started_event.command
```

Comment:
When you use `event.started_event`, what is `event` in that expression?

Reply:
The event is a CommandSucceeded event, which doesn't contain a command.

Reply:
Could you extract that on its own line for clarity please, e.g.

```
started_event = event.respond_to?(:command) ? event : event.started_event
```

Reply:
done